pkg/kubelet/volumes.go

```diff
@@ -134,6 +142,25 @@ func (kl *Kubelet) mountExternalVolumes(pod *api.Pod) (kubecontainer.VolumeMap,
 		if err != nil {
 			return nil, err
 		}
+		if hasFSGroup && builder.SupportsOwnershipManagement() {
```

Comment:
Yeah, I am waiting for final resolution in #15323 and will use the same approach